.github/workflows/documentation-build.yml

```diff
@@ -1,8 +1,10 @@
-# This pipeline 
-# - build html documentation 
-# - deploy html documentation to gh-pages branch of the same repository
+# This pipeline
+# - build developer documentation 
+# - push documentation to gh-pages branch of the same repository
```

Comment:
grammar:
`- builds html documentation`
`- deploys html...`
since this is about the singular pipeline.

Reply:
fixed